WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -203,9 +196,7 @@ private async Task UpdateTransactionAsync(TransactionSummaryViewModel summary, B
 
 	private async Task OnAdjustFeeAsync()
 	{
-		DialogViewModelBase<FeeRate> feeDialog = _info.IsCustomFeeUsed
-			? new CustomFeeRateDialogViewModel(_info)
-			: new SendFeeViewModel(UiContext, _wallet, _info, false);
+		DialogViewModelBase<FeeRate> feeDialog = new SendFeeViewModel(UiContext, _wallet, _info, false);
```

Comment:
nit: use `var` instead of explicit type if the right hand side of the assignment expression is an object creation expression.

```suggestion
		var feeDialog = new SendFeeViewModel(UiContext, _wallet, _info, false);
```